lib/SPIRV/OCLUtil.h

```diff
@@ -135,77 +132,80 @@ struct OCLBuiltinTransInfo {
 //
 ///////////////////////////////////////////////////////////////////////////////
 namespace kOCLBuiltinName {
```

Comment:
[minor] To me, alignment used before looks cleaner.

Reply:
I can disable clang-format for that part, but at least manually fix the alignment to be consistent for all of them, as it is currently not the case.